server/src/main/java/org/elasticsearch/transport/TcpTransport.java

```diff
@@ -999,7 +999,7 @@ protected final void doStop() {
         }
     }
 
-    protected void onException(TcpChannel channel, Exception e) {
+    public void onException(TcpChannel channel, Exception e) {
```

Comment:
hmm why is this public?

Reply:
It is called directly by `Netty4MessageChannelHandler` now, instead of having a separate exception handling method in `Netty4Transport.`